runtime/gc_ctrl.c

```diff
@@ -221,6 +221,7 @@ CAMLprim value caml_gc_set(value v)
        (newminwsz != Caml_state->minor_heap_wsz) are both true,
        the current domain reallocates its own minor heap twice. */
     caml_set_minor_heap_size (newminwsz);
+    CAML_EV_COUNTER (EV_C_FORCE_MINOR_SET_MINOR_HEAP_SIZE, 1);
```

Comment:
Why is this event here rather than inside the body of `caml_set_minor_heap_size`?

Reply:
Good call. I've moved it inside `caml_set_minor_heap_size` and actually made sure it's only emitted when there is a forced minor collection: https://github.com/ocaml/ocaml/pull/11400/commits/9da76f9065ea0d11d9b9280d741338df174ffadd